app/backend/src/tests/test_fixtures.py

```diff
@@ -55,7 +55,11 @@ def db_impl(param):
 
     # drop everything currently in the database
     with session_scope() as session:
-        session.execute("DROP SCHEMA public CASCADE; CREATE SCHEMA public; CREATE EXTENSION postgis;")
+        session.execute(
+            """DROP SCHEMA public CASCADE; CREATE SCHEMA public; CREATE EXTENSION postgis;
+            CREATE EXTENSION fuzzystrmatch;CREATE EXTENSION postgis_tiger_geocoder;CREATE EXTENSION postgis_topology;
```

Comment:
This makes it so the recreated db accurately resembles a fresh db

Reply:
Extensions present in fresh db:
```
postgres=# \dx
                                            List of installed extensions
          Name          | Version |   Schema   |                             Description
------------------------+---------+------------+---------------------------------------------------------------------
 fuzzystrmatch          | 1.1     | public     | determine similarities and distance between strings
 plpgsql                | 1.0     | pg_catalog | PL/pgSQL procedural language
 postgis                | 3.0.3   | public     | PostGIS geometry, geography, and raster spatial types and functions
 postgis_tiger_geocoder | 3.0.3   | tiger      | PostGIS tiger geocoder and reverse geocoder
 postgis_topology       | 3.0.3   | topology   | PostGIS topology spatial types and functions
(5 rows)
```

Reply:
These other extensions are not needed by us, they're just installed in the docker container automatically. Since we don't use them, it just adds time to tests unnecessarily.

Reply:
It executes very fast, I think it would be safer to leave them in to prevent errors from happening in the future. If you really want to I'll take them out.

Reply:
I think we should leave them out until we need them.